src/help/guides/extension-development/cheerio-quickstart.md

```diff
@@ -157,12 +162,15 @@ With a given HTML document, lets do some parsing.
 </ul>
 ```
 
-Given the above set of HTML, say that you have loaded it into Cheerio using the instructions above. What you are looking for, is the text inside each of the `li` objects. There should be a few thoughts which should be going through your head;
+Given the above set of HTML, say that you have loaded it into Cheerio using the instructions above. What you are looking
+for, is the text inside each of the `li` objects. There should be a few thoughts which should be going through your
+head;
 
 - What _unique_ identifier is surrounding the data that I need?
-  - In the example of parsing 'Three Apples', we can see that it is __always__ in a `li` block, with a class of `apple`
+	- In the example of parsing 'Three Apples', we can see that it is __always__ in a `li` block, with a class
```

Comment:
This could be a consequences of the linting but we used to use `*` instead of `-`. I'm not against `-` but I propose we chose which we want to use.
```suggestion
	- In the example of parsing 'Three Apples', we can see that it is **always** in a `li` block, with a class
```